### ISSUE TITLE
Import arya.arya

### ISSUE THREAD

**Issue by jbhoorasingh** [datacenter/webarya#3]
Running the Flask app will get you the below error
`Traceback (most recent call last):
  File "webarya.py", line 5, in <module>
    import arya.arya
ImportError: No module named arya
`
to fix we need to `import arya` not  `import arya.arya`

**Comment by kecorbin** [comment]
Yes there is a (somewhat) known issue related to path here.  There is also a console script that arya implements called `arya` as well.  This can create a namespace colision depending on your path. Are you running in a venv or via docker?
